spec/models/vm_or_template_spec.rb

```diff
@@ -335,6 +335,19 @@
     end
   end
 
+  context "#resource_group" do
+    before do
```

Comment:
If this is a regular activerecord has_one, I don't understand why we'd need to test that.  They're usually helpful when you're developing and want to make sure you did both sides of the association correctly but aren't really needed after that.